docs/schema.md

```diff
@@ -287,6 +287,15 @@ It defines how this object will be handled. Each type has a different set of pro
 
 ---
 
+### type: IMAGE
```

Comment:
This only applies for schema v1. Lets add this session here and have a working item to include schema v2 instructions.

Reply:
Created #387.

Reply:
+1 Thanks for tracking!